documentation/src/docs/asciidoc/extensions.adoc

```diff
@@ -262,6 +262,43 @@ but rethrow any other type of exception.
 include::{testDir}/example/exception/IgnoreIOExceptionExtension.java[tags=user_guide]
 ----
 
+[[extensions-test-templates]]
+=== Providing Invocation Contexts for Test Templates
+
+A `{TestTemplate}` method can only be executed when at least one
+`{TestTemplateInvocationContextProvider}` is registered. Its responsibility is to provide
+a `Stream` of `{TestTemplateInvocationContext}` instances. Each context may provide a
```

Comment:
Its responsibility is to provide --> Each such provider is responsible for providing...